HW1/m_1.py

```diff
@@ -0,0 +1,11 @@
+# найдите сумму цифр трехзнаяного числа
+
+print("введите трехзначное число")
+m = int(input())
```

Comment:
В использовании функции `print` нет никакой необходимости, ведь сообщение для пользователя о вводе мы можем написать прямо внутри функции `input`.
```suggestion
print("введите трехзначное число")
m = int(input("введите трехзначное число "))
```